test/async-hooks/testcfg.py

```diff
@@ -3,4 +3,4 @@
 import testpy
 
 def GetConfiguration(context, root):
-  return testpy.AsyncHooksTestConfiguration(context, root, 'async-hooks')
+  return testpy.ParallelTestConfiguration(context, root, 'async-hooks')
```

Comment:
Actually looking more at this, I think I do object to this part of the change. /cc @nodejs/testing 

I'd prefer leaving the code duplication here. This reduction of code duplication in particular seems to result in magic/surprises and names that are misleading. ("I changed ParallelTestConfiguration and it somehow broke async-hooks tests. Wha?")